README.md

```diff
@@ -1,80 +1,111 @@
 ![Plow logo](./assets/logo.svg)
+
 # Plow - Ontology package manager
 
-Plow is package management solution for OWL ontologies, with support for specifying dependencies between packages via SemVer ranges.
+Plow is package management solution for OWL ontologies, with support for specifying dependencies between packages via [SemVer](https://semver.org/) ranges.
 
 ## Getting started - Installation
 
 ### CLI
-The CLI supports basic commands related to consuming and producing ontologies. It is suitable for both manual and automated workflows (e.g. [metadata linting in CI](https://github.com/field33/ontologies/blob/12ede2b557fde94f6a768e8b65c84929a58c05ce/.github/workflows/lint.yml#L33))
 
-Coming soon: [open in Protégé](https://github.com/field33/plow/issues/10)
+The CLI supports basic commands related to consuming and producing ontologies. It is suitable for both manual and automated workflows (e.g. [metadata linting in CI](https://github.com/field33/ontologies/blob/12ede2b557fde94f6a768e8b65c84929a58c05ce/.github/workflows/lint.yml#L33))
 
 To install, run:
 
-```shell
+```sh
 cargo install plow_cli
 ```
 
 [Prebuilt binaries are coming soon!](https://github.com/field33/plow/issues/1)
 
 ### GUI
 
-Coming soon
+The `plow_gui` crate is obsolete and usage of it is discouraged.
 
-> You can install a preview version with limited functionality via `cargo install plow_cli`
-
-[Prebuilt binaries are coming soon!](https://github.com/field33/plow/issues/2)
+A new ontology editor is in the works, and will be released as a separate project.
 
 ## Basic usage
+
 ### Login with Plow
-The tooling currently expects you to be authenticated with the public plow registry ([open issue](https://github.com/field33/plow/issues/11)). Please sign in, creating an account [here](plow.pm) and [create a new User Token in your account settings](https://staging-registry.field33.com/home#user-tokens).
 
+The tooling currently expects you to be authenticated with the public plow registry ([open issue](https://github.com/field33/plow/issues/11)). Please sign in, creating an account [here](plow.pm) and [create a new api token in your account settings](https://staging-registry.field33.com/home#user-tokens).
```

Comment:
> [here](plow.pm)

It doesn't look like a correct link. it probably should be `https://plow.pm`, but since it redirects to `https://registry.field33.com/` probably the registry link it the one which is preferable, isn't it?

> [create a new api token in your account settings](https://staging-registry.field33.com/home#user-tokens)
I thought stagin is only for internal usage. Shouldn't we use production link here instead?

Reply:
179e7d8